lib/flagon/inspector.rb

```diff
@@ -17,6 +17,13 @@ def when_enabled(flag_name)
       end
     end
 
+    def ensure_flags_exist(*flags)
```

Comment:
Is the convention that we add a '!' when the method could raise an exception? I always forget.